obsbot/cogs/public/log_analysis.py

```diff
@@ -100,6 +100,15 @@ async def on_message(self, msg: Message):
             logger.warning('There are too many possible log URLs, cutting down to 3...')
             log_candidates = log_candidates[:3]
 
+        async def react_with(emoji):
+            try:
+                await msg.add_reaction(emoji)
+            except Exception:
```

Comment:
Would probably be better to catch the specific exceptions, which are `discord.Forbidden` and `discord.NotFound`

`except (discord.Forbidden, discord.NotFound):`